Userland/Libraries/LibSQL/AST/Insert.cpp

```diff
@@ -11,6 +12,19 @@
 
 namespace SQL::AST {
 
+static bool does_value_data_type_match(const SQLType expected, const SQLType actual)
+{
+    if (actual == SQLType::Null) {
+        return false;
+    }
```

Comment:
```suggestion
    if (actual == SQLType::Null)
        return false;
```

The style in SerenityOS is to not add braces for single line statements.
Although I would like to point to https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md it is actually not mentioned there (perhaps it should?).

Reply:
Apparently this is not as strict as I thought you can leave them as long as the code is consistent (so the whole file either does or doesn't)

Reply:
@davidot Alright, I guess I'll leave my braces on since the original commit of the file had braces.